config.yaml

```diff
@@ -1,5 +1,20 @@
 - destination: &dest syn12177492
 - datasets:
+    - biodomains:
+        files:
+          - name: biodomains
+            id: syn44151254.1
+            format: csv
+        final_format: json
+        custom_transformations: 1
+        provenance:
+          - syn44151254.1
+        column_rename:
+          goterm_name: go_terms
+        agora_rename:
+          ensembl_id: ensembl_gene_id
```

Comment:
Is there a reason that this is an `agora_rename` instead of a `column_rename`? Most other datasets appear to be using `column_rename` to standardize to `ensembl_gene_id` before the transform.

Reply:
Nope! Just confusion on my part...I've updated the aliasing and transform to use ensembl_gene_id now.